### ISSUE TITLE
can't install CCX file. Photoshop 2024

### ISSUE THREAD

**Issue by kubilaykilinc** [NimaNzrii/comfyui-photoshop#41]
I am using Photoshop 2024. But i can't install ccx file with ZPX installer. I've got error message.

**Comment by discovermodo** [comment]
@kubilaykilinc You can simply create a folder inside Photoshop 2024 > Plug-ins folder (ex: ComfyUI Photoshop) and copy/extract files inside .ccx file to there ( .ccx file is actually a zip file. you can simply change the file extension from .ccx to .zip and open it.)

**Comment by NimaNzrii** [comment]
Try this: login to your adobe creative cloud

And try again installing the ccx file

**Comment by bubbl000** [comment]
cxx to zip Then put the files in the archive into the folder Folder moved to C:Program FilesAdobeAdobe Photoshop 2023Plug-ins

**Comment by kubilaykilinc** [comment]
I am getting this error 
![image](https://github.com/NimaNzrii/comfyui-photoshop/assets/18080130/6e0f3d9e-85e7-46d6-b885-b996321a6b44)

**Comment by kamiri007** [comment]
For others who can't install it
 https://www.youtube.com/watch?v=BY9-3mhR3Vc

**Comment by kubilaykilinc** [comment]
It is working. Thank you. 
![image](https://github.com/NimaNzrii/comfyui-photoshop/assets/18080130/e09e9344-ec36-463b-bab6-211fd8fc5a28)

**Comment by bubbl000** [comment]
> For others who can't install it对于无法安装它的其他人 https://www.youtube.com/watch?v=BY9-3mhR3Vc

How do I put the generated image from the plugin into the canvas??? Then why can't my comfyui read the canvas width and height canvas content

**Comment by kubilaykilinc** [comment]
> > For others who can't install it对于无法安装它的其他人 https://www.youtube.com/watch?v=BY9-3mhR3Vc
> 
> How do I put the generated image from the plugin into the canvas??? Then why can't my comfyui read the canvas width and height canvas content

I'm just in the process of discovering this workflow. Therefore, after seeing the plug-in in Photoshop, I simply did the following.
1- I ran ComfyUI. I downloaded the workflow shared in the CODE section and opened it in comfyUI. However, I haven't run the image generation process yet.
2- I opened Photoshop and ran the plug-in.
3- This video actually explains the process. https://www.youtube.com/watch?v=JwLbnO4px-E&ab_channel=AIwithNima
